version/terraform_test.go

```diff
@@ -31,7 +31,7 @@ func TestTerraformConstraint(t *testing.T) {
 			false,
 		}, {
 			"invalid upper bound",
-			"0.16.2",
+			"1.1.0",
```

Comment:
I saw the other versions have a "valid <version>" test case. Wondering what you think about including a "valid 1.0" test case as well?

Reply:
Sure thing, forgot about that one!